addons/web/static/src/views/fields/properties/property_value.scss

```diff
@@ -18,17 +18,38 @@ input.o_field_property_input[type=number] {
     height: 25px;
 }
 
+.o_field_property_many2one_value {
+    img {
+        height: 1.4em;
+        width: 1.4em;
+        object-fit: cover;
+    }
+
+    &.avatar .o_input_dropdown{
+        // counter the avatar size to keep things aligned
+        margin-left: -1.6em;
+        padding-left: 1.6em;
+    }
+}
+
 .o_field_property_many2many_value {
-    .o_tag {
-        margin: 1px 2px 1px 0;
-        .o_delete {
-            padding-left: 4px;
-        }
+    .o_tag,
+    .o_tag_badge_text,
+    .o_delete {
+        color: gray !important;
```

Comment:
hardcoded gray ? Don't we have some standard variable-based colors for that ? Just asking